src/ripple/rpc/impl/LookupLedger.cpp

```diff
@@ -34,15 +35,16 @@ bool isValidatedOld ()
         Tuning::maxValidatedLedgerAge;
 }
 
-Status ledgerFromRequest (
-    Json::Value const& params,
-    Ledger::pointer& ledger,
-    NetworkOPs& netOps)
+template <class T>
+Status ledgerFromRequest (T& ledger, Context& context)
```

Comment:
I suspect a better signature is
`Status ledgerFromRequest (ReadView const& ledger, Context& context)`